2.7.0/todo/result_onnx.md

```diff
@@ -26,15 +25,23 @@ The categories below are as follows:
 * not user facing: All commits that are not public end-user facing and hence should be dropped from the release notes
 
 ## onnx
+
 ### bc breaking
+
 - [ONNX] Set USE_EXPERIMENTAL_LOGIC to True ([#137296](https://github.com/pytorch/pytorch/pull/137296))
+
 ### deprecation
+
 - [ONNX] Create deprecation warning on dynamo_export ([#146003](https://github.com/pytorch/pytorch/pull/146003))
 - [ONNX] Create deprecation warning on dynamo_export ([#146425](https://github.com/pytorch/pytorch/pull/146425))
 - [ONNX] Adjust and add deprecation messages ([#146639](https://github.com/pytorch/pytorch/pull/146639))
+
 ### new features
+
 - [ONNX] Support custom axis name through dynamic_shapes ([#146321](https://github.com/pytorch/pytorch/pull/146321))
```

Comment:
Please add backticks for code to make them more easily readable, following the instructions here: https://docs.google.com/document/d/14OmgGBr1w6gl1VO47GGGdwrIaUNr92DFhQbY_NEk8mQ/edit?tab=t.0#heading=h.2f3oh5cws8m8